Is_Binary_Search_Tree/Is_Binary_Search_Tree.java

```diff
@@ -0,0 +1,86 @@
+    
+    import java.util.Scanner;
```

Comment:
4 space indentation should be given as we go inside blocks. Please remove the extra 4 space added at start of line throughout the file.